selfdrive/test/process_replay/process_replay.py

```diff
@@ -447,6 +538,24 @@ def _replay_single_process(cfg, lr, fingerprint, disable_progress):
       return log_msgs
 
 
+def setup_vision_ipc(cfg, lr):
+  assert len(cfg.vision_pubs) != 0
+
+  device_type = next(msg.initData.deviceType for msg in lr if msg.which() == "initData")
+  is_eon = device_type == "eon"
+
+  vipc_server = VisionIpcServer("camerad")
+  if "roadCameraState" in cfg.vision_pubs:
+    vipc_server.create_buffers(VisionStreamType.VISION_STREAM_ROAD, 40, False, *(eon_f_frame_size if is_eon else tici_f_frame_size))
```

Comment:
no need for 40 buffers here, it'll just use more memory unnecessarily

Reply:
now it's 5

Reply:
should only need one or maybe two since everything's blocking